index.html

```diff
@@ -13,9 +13,20 @@
 </head>
 
 <body>
+    <h1>Welcome To Confession Page</h1>
     <div id="root"></div>
+    <div class="container inputarea">
+      <fieldset>
+      <legend>Enter details for confessions</legend>
+      <h1>Name of the person you want to confess</h1>
+      <input class="card" type="text" name="name" placeholder="Name" autocomplete="off">
+      <h1>Write your confession here</h1>
+      <textarea class="card" name="name" maxlength="200" placeholder="Enter confess in 200 max words........."></textarea>
+      <button type="submit" name="button">Submit</button>
```

Comment:
Can we beautify, this button?

Reply:
yaaa i forgot to beautify it. i will do that.